environment.yml

```diff
@@ -11,13 +11,9 @@ dependencies:
  - cftime>=1.0.4
  - netCDF4>=1.4
  - poppler>=0.67
- - fiona>=1.8
- - geojson>=2.5.0
- - shapely>=1.6
- - geopandas>=0.7
+ - rtree>=0.9.4
```

Comment:
If `geopandas` is not installed, does xclim still need the rtree library?

Reply:
Also, is the newest clisops available as a conda-forge package?

Reply:
clisops conda-forge package is available (currently building). I don't know about the `rtree` ... I remove it and run the tests.

Reply:
removed `rtree` ... cleaned requirements in `setup.py`. tests are still working (locally).

Reply:
We might also be able to remove the `spatialindex` and `libspatialindex-dev` apt/homebrew libraries from the travis-ci builds at the same time. Those are only there to support rtree and GeoPandas.